packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts

```diff
@@ -44,7 +44,7 @@ test.describe('AtomicTabManager', () => {
       test.beforeEach(async ({facets}) => {
         await facets.getFacetValue.first().waitFor({state: 'visible'});
       });
-      test('facets', async ({tabManager}) => {
+      test.fixme('facets', async ({tabManager}) => {
```

Comment:
When will those tests be addressed ?

Reply:
@alexprudhomme in https://github.com/coveo/ui-kit/pull/4313